swagger.yaml

```diff
@@ -857,10 +856,10 @@ paths:
       tags:
         - VacationRequests
       parameters:
-        - name: personId
+        - name: userId
           required: false
```

Comment:
Does this false need to be here ? Can it be removed if nothing is required ?

Reply:
Or actually maybe it should be required...